serviceworker/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<!--
+*
+* Copyright (C) 2016, bitmovin GmbH, All Rights Reserved
+*
+* This source code and its use and distribution, is subject to the terms
+* and conditions of the applicable license agreement.
+*
+-->
+<html lang="en">
+<head>
+  <title>Bitmovin Player Demo</title>
+  <meta charset="UTF-8"/>
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+  <!-- Bitmovin Player -->
+  <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
+  <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/modules/bitmovinplayer-serviceworker-client.js"></script>
+
+  <style>
+    .container {
+      color: white;
+      text-align: center;
+    }
+    .container a {
+      color: white;
+    }
+    .container h1 {
+      margin-bottom: 22px;
+      line-height: 66px;
+    }
+    .container h2 {
+      font-weight: normal;
+      margin-bottom: 36px;
+      line-height: 26px;
+    }
+    .player-wrapper {
+      width: 95%;
+      margin: 20px auto;
+      box-shadow: 0 0 30px rgba(0,0,0,0.7);
+    }
+  </style>
+</head>
+<body>
+<div class="container">
+  <div class="content">
+    <div class="player-wrapper">
+      <div id="player"></div>
+    </div>
+
+    <div class="player-wrapper">
+      <div id="player2"></div>
+    </div>
+  </div>
+</div>
+<script type="text/javascript">
+  function unregisterAllServiceWorker() {
+    return navigator.serviceWorker.getRegistrations().then((registrations) => {
+      return Promise
+      .all(registrations.map(registration => registration.unregister()))
+      .then(() => {});
+    });
+  }
+
+  const conf = {
+    key: 'YOUR KEY HERE',
+    style: {},
+    cast: {
+      enable: true
+    },
+    events: {
+      segmentplayback: console.warn,
+      metadata: console.warn,
+      metadataparsed: console.warn,
```

Comment:
maybe just log?

Reply:
Just `log` is not so nice in safari. I would prefer to have these events as `warn` to be more visible in the console.

Reply:
OK